.github/workflows/test-origami-component.yml

```
@@ -5,18 +5,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    # Required for npx origami-ci branch, specifically for bundle-size-cli
-    - run: git fetch --prune --unshallow
     - uses: actions/setup-node@v2.1.5
       with:
         node-version: 12.x
-    - run: npm install --only=dev
-    - name: Repository name
-      id: repo_name
-      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
-    - run: npx origami-ci branch
-      env:
-        ORIGAMI_CI_NAME: ${{ steps.repo_name.outputs.REPO_NAME }}
-        ORIGAMI_CI_VERSION: 0.0.0
-        GITHUB_TOKEN: ${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}
-        ORIGAMI_PULL_REQUEST: ${{ github.event.pull_request.html_url }}
+    - run: npm install -g npm@^7
+    - run: npm install -g origami-build-tools@v11.0.0-beta.18
```

If we use the `prerelease` dist-tag instead of a specific version tag then it will always use the latest beta version and won't need to keep updating this when we release new v11 betas. This was a suggestion from [past @chee in Slack](https://financialtimes.slack.com/archives/CSW6B2VAN/p1617271670047000) :-D

```suggestion
    - run: npm install -g origami-build-tools@prerelease
```

---

lol yep, this line is actually exactly why i brought that up